verification/curator-service/api/src/geocoding/mapbox.ts

```diff
@@ -44,6 +46,15 @@ function getResolution(features: GeocodeFeature[]): Resolution {
     return Resolution.Country;
 }
 
+// Get the mapbox types for a given Resolution.
+const resolutionToGeocodeQueryType = new Map<Resolution, GeocodeQueryType>([
+    [Resolution.Country, 'country'],
+    [Resolution.Admin1, 'region'],
+    [Resolution.Admin2, 'district'],
+    [Resolution.Admin3, 'place'],
+    [Resolution.Point, 'poi'],
```

Comment:
Did we call this (our version of mapbox's poi) place in this data service def?

Reply:
There is no "place" resolution in the schema, it's "point".
Happy to rename if we prefer this.

Reply:
Gotcha -- couldn't remember. All good.